openquake/calculators/tests/outputs/sigma_epsilon.csv

```diff
@@ -1,3 +1,3 @@
-event_id,sig,eps
-0,2.600000E-01,-1.842593E+00
-1,2.600000E-01,3.478994E-01
+event_id,rlz_id,sig_inter_PGA,eps_inter_PGA
+10,1,NAN,NAN
```

Comment:
Are the NAN values expected?

Reply:
Yes, this happens for GMPE not defined for "inter" but only for "total".